lib/ansible/modules/windows/win_domain_user.ps1

```diff
@@ -112,6 +114,31 @@ If ($state -eq 'present') {
         $result.changed = $true
     }
 
+    # Configure password policies
+    If (($null -ne $password_never_expires) -and ($password_never_expires -ne $user_obj.PasswordNeverExpires)) {
```

Comment:
What's happening here? Seems like it should be in a separate PR or just add a changelog entry.

Reply:
HAd some issues with this after you merged the removal of the try from this. - the indentations were off and the if block was not closed - ill give it a second look against the devel current file

Reply:
Fixed

Reply:
Ahh cool so it was just a rebase gone wrong and not a problem with my PR?

Reply:
lol yeah